### DISCUSSION TITLE
Runtime Integrity Checks on iOS

### DISCUSSION THREAD

**Discussion by serrapa** [OWASP/mastg#2470] [Q&A] [answered]
Can I ask why there is no MSTG test for the Runtime Integrity Checks on iOS ? (For android there is)

**Comment by cpholguera** [comment, marked as answer]
The MASTG is a resource that grows with time and the help of all volunteers. Unfortunately, full coverage can't be guaranteed at the moment.

We're currently moving to v2 and adding new tests.

Keep an eye out for the following weakness (currently in draft)

https://mas.owasp.org/MASWE/MASVS-RESILIENCE/MASWE-0107/

If you have experience in this area and would like to contribute, please let us know. Feel free to open a PR to complete the new weakness and/or create the relevant tests and demos.

More info: 
- https://mas.owasp.org/news/
- https://mas.owasp.org/talks/
